vpn-router-core/src/main/java/vpnrouter/core/infrastructure/shell/ShellExecutor.java

```diff
@@ -0,0 +1,34 @@
+package vpnrouter.core.infrastructure.shell;
+
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import vpnrouter.core.exception.UserException;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+@Component
+public class ShellExecutor {
+
+    @SneakyThrows
+    public List<String> execute(String command) {
+        var process = new ProcessBuilder("sh", "-c", command).start();
```

Comment:
wow! I haven't known about such things before)

Reply:
Hm. You can start subprocesses from Java code :)